vaadinfrontend/src/main/java/life/qbic/datamanager/views/general/funding/FundingField.java

```diff
@@ -45,6 +34,10 @@ private static Div layoutFundingInput(TextField label, TextField referenceId) {
     return layout;
   }
 
+  private void layoutComponent() {
```

Comment:
You could inline this method.
Something like `add(layoutInputs)` is readable enough and when looking for the layout can be detected directly.